.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,56 @@
+name: Python application
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  lint-flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Lint with flake8
+        run: |
+          pip install flake8
+          flake8 .
+  lint-black:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Lint with black
+        run: |
+          pip install black
+          black --check .
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Test with pytest
+        run: |
+          pip install pytest
+          python setup.py develop
```

Comment:
What is the purpose of this line of code?

Reply:
It makes the plugin module available in the environment? Something like that. @matburt Care to help me here, I'm not realloy a python person

Reply:
Ahhh, yeah, I see why this is here. It installs the code in develop/editable mode, which is needed for stuff like imports to resolve. It's just a much less common technique than `pip install -e .` or similar.

Reply:
Should I change it to use `pip`?

Reply:
AFAIK, the end result is about the same. This seems fine.